iree/jax/program_api.py

```diff
@@ -416,69 +416,69 @@ def __new__(mcls,
     for key in remove_keys:
       del dct[key]
 
-    # Hide any static attributes defined on the Module class. We only want them
-    # to be accessible as `Module.foo` and not to pollute subclass namespaces.
+    # Hide any static attributes defined on the Program class. We only want them
+    # to be accessible as `Program.foo` and not to pollute subclass namespaces.
     # This must be done after we process user specified attributes to hide
     # any remaining.
-    for key in _STATIC_MODULE_ATTRIBUTES:
+    for key in _STATIC_PROGRAM_ATTRIBUTES:
       if key not in dct:
         dct[key] = _hide_instance_attribute
 
     # Attach the info instance.
     new_class = type.__new__(mcls, name, bases, dct)
-    _module_class_infos[new_class] = info
+    _program_class_infos[new_class] = info
     return new_class
 
   def __getattr__(cls, key):
-    # The Module base class has no info object and does not need dynamic
+    # The Program base class has no info object and does not need dynamic
     # resolution.
-    if cls is Module:
-      raise AttributeError(f"Module.{key}")
-    info = Module.get_class_info(cls)
+    if cls is Program:
+      raise AttributeError(f"Program.{key}")
+    info = Program.get_class_info(cls)
     try:
       return info.all_exports[key]
     except KeyError:
       raise AttributeError
 
 
-_STATIC_MODULE_ATTRIBUTES = (
+_STATIC_PROGRAM_ATTRIBUTES = (
     "_get_instance",
     "export_global",
     "get_class_info",
     "get_compiled_artifact",
     "get_info",
-    "get_mlir_module",
+    "get_mlir_program",
```

Comment:
So, in this case, this actually is returning an "miles module". Let's keep this one as is.

Reply:
Reverted